modules/internal/ChapelRange.chpl

```diff
@@ -406,12 +406,94 @@ module ChapelRange {
 
   pragma "last resort"
   proc chpl_compute_low_param_loop_bound(param low, param high) param {
-    compilerError("Range bounds must be integers of compatible types");
+    compilerError("Range bounds must be integers of compatible types in param for-loops");
   }
 
   pragma "last resort"
   proc chpl_compute_low_param_loop_bound(low, high) {
-    compilerError("param for loop must be defined over a bounded param range");
+    compilerError("param for-loops must be defined over a bounded param range");
```

Comment:
I think if you do a `git grep` for this message, you will find existing tests whose .good files need updating.

Reply:
Yes, sir.